### PR TITLE
fixing delete issue in Tpcc scenario

### PR DESCRIPTION
**Pull request by dbhatnagar-px**
Tpcc test case is passing but it failed in DeletePvandPVCs. This PR is a fix for that.

### CODE REVIEW

**Review by Madanagopal19 — APPROVED** [review]
LGTM

**Review by aravindranath14 — APPROVED** [review]
Looks good

**Review by Madanagopal19 — COMMENTED** [review]
I discussed with @dbhatnagar-px few more changes are required, lets fix it and merge this PR

File: tests/pds/dataservice_test.go [hunk 1]
```diff
@@ -758,17 +761,6 @@ func deployAndTriggerTpcc(dataservice, Version, Image, dsVersion, dsBuild string
 				}
 			}
 		})
-
-		Step("Delete Deployments", func() {
```
Comment (Madanagopal19) [inline tests/pds/dataservice_test.go:762]: removing this part will delete only the latest deployment

**Review by Madanagopal19 — CHANGES_REQUESTED** [review]

File: tests/pds/dataservice_test.go (hunk 1 above)
Comment (dbhatnagar-px) [reply tests/pds/dataservice_test.go:763]: Done

**Review by Madanagopal19 — APPROVED** [review]
lgtm

**Pull request merged**